src/ROUTES.ts

```diff
@@ -296,7 +296,7 @@ const ROUTES = {
     REPORT: 'r',
     REPORT_WITH_ID: {
         route: 'r/:reportID?/:reportActionID?',
-        getRoute: (reportID: string, reportActionID?: string, referrer?: string) => {
+        getRoute: (reportID: string | undefined, reportActionID?: string, referrer?: string) => {
```

Comment:
I think this method shouldn't be called without `reportID` at all, so you can try to add condition on a level higher. Similar discussion: https://github.com/Expensify/App/pull/54306#discussion_r1890761763

This way you can remove all of the updates in this file.